dts/arm/st/f2/stm32f207.dtsi

```diff
@@ -10,17 +10,28 @@
 	soc {
 		compatible = "st,stm32f207", "st,stm32f2", "simple-bus";
 
-		mac: ethernet@40028000 {
-			compatible = "st,stm32-ethernet";
+		ethernet@40028000 {
 			reg = <0x40028000 0x8000>;
-			interrupts = <61 0>;
-			clock-names = "stmmaceth", "mac-clk-tx",
-				      "mac-clk-rx", "mac-clk-ptp";
-			clocks = <&rcc STM32_CLOCK(AHB1, 25U)>,
-				 <&rcc STM32_CLOCK(AHB1, 26U)>,
-				 <&rcc STM32_CLOCK(AHB1, 27U)>,
-				 <&rcc STM32_CLOCK(AHB1, 28U)>;
-			status = "disabled";
+			compatible = "st,stm32-ethernet-parent";
+			clock-names = "stmmaceth";
```

Comment:
Nit: if this clock is remove from mac, then following would be more suited:
```suggestion
			clock-names = "stm-eth";
```
This would likely require driver update.

Reply:
changed in https://github.com/zephyrproject-rtos/zephyr/pull/87593

Reply:
:-D commented in the wrong PR